module/mfrc522_command.c

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: GPL-2.0
+
+#include "mfrc522_command.h"
+#include "linux/kernel.h"
+#include "linux/slab.h"
+#include "linux/string.h"
+#include "mfrc522_spi.h"
+
+struct mfrc522_command *mfrc522_command_init(u8 cmd, char *data, u8 data_len)
+{
+	struct mfrc522_command *command =
+		kmalloc(sizeof(struct mfrc522_command), GFP_KERNEL);
```

Comment:
Seeing how `mfrc522_command` is used, do we really need to allocate a struct for it ?